common/build.gradle

```diff
@@ -169,6 +169,10 @@ dependencies {
     testImplementation 'org.json:json:20160810'
     testImplementation "junit:junit:$rootProject.ext.junitVersion"
     testImplementation "org.mockito:mockito-core:$rootProject.ext.mockitoCoreVersion"
+
+    testImplementation("org.mockito.kotlin:mockito-kotlin:4.1.0") {
```

Comment:
The version numbers have to be provided here for the PR build to succeed. To use a variable from the gradle/versions.gradle file I will have to modify the dev branch of ADAL, MSAL and Broker repositories. I will do that sometime in October.